lib/internal/timers.js

```diff
@@ -556,7 +555,7 @@ function getTimerCallbacks(runNextTicks) {
         if (args === undefined)
           timer._onTimeout();
         else
-          ReflectApply(timer._onTimeout, timer, args);
+          timer._onTimeout.apply(timer, args);
```

Comment:
This will need to go back to using `primordials.ReflectApply` to fix the falling test. Since `primordials.ReflectApply === globalThis.Reflect.apply`, it would be very surprising if it had any performance impact anyway.

Reply:
Note that `Reflect.apply()` is different from `func.apply()`.